ui/src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import { useObserver } from 'mobx-react';
+import { Text, TextVariants } from '@patternfly/react-core';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { Button } from '@patternfly/react-core/dist/js/components';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import '@patternfly/react-core/dist/styles/base.css';
```

Comment:
I would suggest to move this to the top level of this app, so it can be applied everywhere instead of using the base css in the component files directly.